sdcflows/interfaces/utils.py

```diff
@@ -384,3 +404,40 @@ def _reoblique(in_epi, in_plumb, in_field, in_mask=None, newpath=None):
         masknii.__class__(masknii.dataobj, epinii.affine, hdr).to_filename(out_files[2])
 
     return out_files
+
+
+def _pad_num_slices(in_file, newpath=None):
+    """
+    Check if image has an even number of slices.
+    If it does, return the image unaltered.
+    Otherwise, return the image with an empty slice added.
+
+    Parameters
+    ----------
+    img : :obj:`str` or :py:class:`~nibabel.spatialimages.SpatialImage`
+        3D or 4D NIfTI image
+
+    Returns
+    -------
+    file : :obj:`str`
+        The output file with even number of slices
+    padded : :obj:`bool`
+        Indicator if the input image was padded.
+
+    """
+    import nibabel as nb
+    from nipype.utils.filemanip import fname_presuffix
+    import numpy as np
+
+    img = nb.load(in_file)
+    if img.shape[2] % 2 == 0:
```

Comment:
With rodents, we may want to allow another axis. May we use add a default argument `ax=2` to the signature instead?
```suggestion
    if img.shape[ax] % 2 == 0:
```